packages/databricks-sdk-js/src/apis/workspace/model.ts

```diff
@@ -46,13 +46,7 @@ export interface Export {
  * `SOURCE`. However it may be one of: `SOURCE`, `HTML`, `JUPYTER`, `DBC`. The
  * value is case sensitive.
  */
-export type ExportFormat =
-    | "AUTO"
-    | "DBC"
-    | "HTML"
-    | "JUPYTER"
-    | "R_MARKDOWN"
-    | "SOURCE";
+export type ExportFormat = "DBC" | "HTML" | "JUPYTER" | "R_MARKDOWN" | "SOURCE";
```

Comment:
Just need to revert this for manual update for now.

Reply:
I was wondering about it already. Do you know how to get this change upstream?

Reply:
Done